files/zuul-pre-commit.yaml

```diff
@@ -12,18 +12,14 @@
         name:
           - git-core
           - python36
-      become: true
-    - name: Install pre-commit PyPI package
-      pip:
-        name: pre-commit
+          - pre-commit
       become: true
       tags:
         - stop-layering
-    - name: Install pre-commit for the repo
+    - name: Run 'pre-commit install' to set up git hook scripts
```

Comment:
Is this needed, does it also install the hook requirements?

We don't need the git hooks.

Reply:
I don't know. Removed.